### ISSUE TITLE
Removing a Connected Class Caused a JuggleError

### ISSUE THREAD

**Issue by danyeaw** [gaphor/gaphor#595]
**Describe the bug**
Trying to disconnecting the tail of a connected generalization that was connected to a stereotype resulted in the following JuggleError in Gaphas:

```
Traceback (most recent call last):
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/decorators.py", line 139, in async_wrapper
    func(*args, **kwargs)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/view/gtkview.py", line 310, in update
    model.update_now(dirty_items, dirty_matrix_items)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/decorators.py", line 169, in wrapper
    return func(*args, **kwargs)
  File "/home/dan/Projects/gaphor/gaphor/core/modeling/diagram.py", line 374, in update_now
    self._connections.solve()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/connections.py", line 74, in solve
    self._solver.solve()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/solver.py", line 186, in solve
    c.solve()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/constraint.py", line 107, in solve
    self.solve_for(wvar)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/constraint.py", line 449, in solve_for
    self._solve()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/constraint.py", line 475, in _solve
    _update(px, x)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/constraint.py", line 102, in _update
    variable.value = value
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/state.py", line 61, in wrapper
    return func(*args, **kwargs)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/variable.py", line 113, in set_value
    self.dirty()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/variable.py", line 105, in dirty
    self.notify()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/variable.py", line 91, in notify
    handler(self)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/constraint.py", line 73, in _propagate
    self.notify()
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/constraint.py", line 69, in notify
    handler(self)
  File "/home/dan/Projects/gaphor/.venv/lib/python3.9/site-packages/gaphas/solver/solver.py", line 140, in request_resolve_constraint
    raise JuggleError(
gaphas.solver.solver.JuggleError: Variable juggling detected, constraint <gaphas.constraint.LineConstraint object at 0x7fe00b2abdf0> resolved 188 times out of 646
```

Trying to save and reopen the model resulted in a failure to load the model and the following error:

```
Traceback (most recent call last):
  File "/home/dan/Projects/gaphor/gaphor/ui/actiongroup.py", line 138, in _action_activate
    method(from_variant(param))
  File "/home/dan/Projects/gaphor/gaphor/ui/appfilemanager.py", line 123, in action_open_recent
    self.load(filename)
  File "/home/dan/Projects/gaphor/gaphor/ui/appfilemanager.py", line 51, in load
    file_manager.load(filename)
  File "/home/dan/Projects/gaphor/gaphor/ui/filemanager.py", line 109, in load
    worker.reraise()
  File "/home/dan/Projects/gaphor/gaphor/ui/gidlethread.py", line 115, in reraise
    raise exc
  File "/home/dan/Projects/gaphor/gaphor/ui/gidlethread.py", line 119, in __generator_executer
    result = next(self._generator)
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 300, in load_generator
    for percentage in load_elements_generator(
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 181, in load_elements_generator
    elem.element.postload()
  File "/home/dan/Projects/gaphor/gaphor/diagram/presentation.py", line 264, in postload
    self._handles.insert(1, self._create_handle(p0))
AttributeError: 'GeneralizationItem' object has no attribute '_create_handle'
```

**To Reproduce**
1. Open the Safety.gaphor model
2. Open the Profiles, General, General Concepts Library, Cause diagram
3. Try to move the tail handle off of the bottom Cause stereotype

**Expected behavior**
No errors.

**Screenshots**

**OS**
- [X] Linux
- [ ] MacOS
- [ ] Windows

NOTES:

**Version**
Version of Gaphor: Master

**Additional context**

**Comment by amolenaar** [comment]
I can not reproduce this error with PR #588.

**Comment by danyeaw** [comment]
I can't either, closed by #588.
